mopidy/core/mixer.py

```diff
@@ -21,7 +22,9 @@ def get_volume(self):
 
         The volume scale is linear.
         """
-        if self._mixer is not None:
+        if self._mixer is None:
+            return -1
```

Comment:
The behavior here no longer matches the docstring. Why not continue to return `None` if the volume is unknown?